src/core_plugins/kibana/public/discover/index.html

```diff
@@ -25,25 +25,19 @@
       >
         <div class="typeahead" kbn-typeahead="discover">
           <div class="kuiLocalSearch">
+            <div class="kuiLocalSearch__icon kuiIcon fa-search"></div>
             <input
               parse-query
               input-focus
               kbn-typeahead-input
               ng-model="state.query"
-              placeholder="Search..."
+              placeholder="Search... ex. status:200 AND extension:PHP"
               aria-label="Search input"
               type="text"
-              class="kuiLocalSearchInput"
+              class="kuiLocalSearch__input"
               ng-class="{'kuiLocalSearchInput-isInvalid': discoverSearch.$invalid}"
             >
-            <button
-              type="submit"
-              aria-label="Search"
-              class="kuiLocalSearchButton"
-              ng-disabled="discoverSearch.$invalid"
-            >
-              <span class="fa fa-search" aria-hidden="true"></span>
-            </button>
+            <a class="kuiLink kuiLocalSearch__help" ng-href="{{queryDocLinks.luceneQuerySyntax}}" target="_blank">Uses lucene query syntax</a>
```

Comment:
According to our HTML style guide, we can improve readability by breaking this up into multiple lines:

```html
<a
  class="kuiLink kuiLocalSearch__help"
  ng-href="{{queryDocLinks.luceneQuerySyntax}}"
  target="_blank"
>
  Uses lucene query syntax
</a>
```